dolphin/tests/unit/api/fakes.py

```diff
@@ -38,3 +38,41 @@ def blank(cls, *args, **kwargs):
         out.environ['dolphin.context'] = context.RequestContext(
             is_admin=use_admin_context)
         return out
+
+
+def fake_storage_pool_get_all(context, marker=None,
+                              limit=None, sort_keys=None,
+                              sort_dirs=None, filters=None, offset=None):
+    return [
+        {
+            "created_at": "2020-06-10T07:17:08.707356",
+            "updated_at": "2020-06-10T07:17:08.707356",
+            "id": "14155a1f-f053-4ccb-a846-ed67e4387428",
+            "name": "SRP_1",
+            "storage_id": '12c2d52f-01bc-41f5-b73f-7abf6f38a2a6',
+            "original_id": "SRP_1",
+            "description": "Dell EMC VMAX Pool",
```

Comment:
suggest use fake storage replace vendor in the unittest. plz update all.

Reply:
done